playbooks/install-vddk.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: CFME | Install VDDK
+  hosts: all
```

Comment:
Should hosts be narrowed to the just the appliances? `cfme-appliances:!cfme-databases`

Or should the databases have this installed as well?

Reply:
Actually, I debated doing something different there but was undecided.  You truly only need to install this on workers in the vCenter zone with Smart State Analysis enabled.